flutter.el

```diff
@@ -119,16 +119,29 @@ ARGS is a space-delimited string of CLI flags passed to
       ,@body)))
 
 (defun flutter--running-p ()
-  "Return non-nil if the `flutter` process is already running."
+  "Return non-nil if the `flutter run` process is already running."
   (comint-check-proc flutter-buffer-name))
 
 (defun flutter--send-command (command)
   "Send COMMAND to a running Flutter process."
   (flutter--with-run-proc
+   flutter-buffer-name
    nil
    (let ((proc (get-buffer-process flutter-buffer-name)))
      (comint-send-string proc command))))
 
+(defun flutter--test (&optional args)
+  "Execute `flutter test` inside Emacs.
+
+ARGS is a space-delimited string of CLI flags passed to
+`flutter`, and can be nil.  Call with a prefix to be prompted for
+args."
+  (interactive
+   (list (when current-prefix-arg
+           (read-string "Args: "))))
+  (flutter--from-project-root
+   (compilation-start (format "%s test %s" (flutter-build-command) args) t)))
```

Comment:
The function docstring says that `args` can be nil, but the resulting command string will be `flutter test nil`. I don't think that's what you want.

Reply:
Thanks, I wrapped with an `or`